### PR TITLE
MGMT-2573 Use the mco image passed from assisted-service

### PR DESCRIPTION
**Pull request by sacharya**

### CODE REVIEW

**Comment by sacharya** [comment]
This seems to be the only place where OpenshiftVersion is used: https://github.com/openshift/assisted-installer/blob/1f4c9863f68c3d67cfa4a67c7da3e3e674ce8f00/src/utils/openshift_version.go#L9

With the MCO Image coming from assisted-service, we should move this validation to the assisted-service side. I am thinking of lookng at the version in the release info for OPENSHIFT_INSTALL_RELEASE_IMAGE, and validate the major version is something we support. Thoughts?

```
oc adm release info quay.io/openshift-release-dev/ocp-release@sha256:2c22e1c56831935a24efb827d2df572855ccd555c980070f77c39729526037d5
Name:      4.6.0-rc.4
Digest:    sha256:2c22e1c56831935a24efb827d2df572855ccd555c980070f77c39729526037d5
Created:   2020-10-13T19:57:04Z
OS/Arch:   linux/amd64
Manifests: 444

Pull From: quay.io/openshift-release-dev/ocp-release@sha256:2c22e1c56831935a24efb827d2df572855ccd555c980070f77c39729526037d5

Release Metadata:
  Version:  4.6.0-rc.4
  Upgrades: 4.5.11, 4.5.13, 4.5.14, 4.5.15, 4.6.0-rc.2, 4.6.0-rc.3
  Metadata:
    description: 
  Metadata:
    url: https://access.redhat.com/errata/RHBA-2020:4196
```

**Comment by sacharya** [comment]
See assisted-service PR: https://github.com/openshift/assisted-service/pull/648

File: src/config/config.go [hunk 1]
```diff
@@ -18,6 +18,11 @@ type Config struct {
 	URL                  string
 	Verbose              bool
 	OpenshiftVersion     string
+<<<<<<< HEAD
```
Comment (djzager) [inline src/config/config.go:21]: Looks like you have some leftovers from a rebase

**Review by eranco74 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**